src/vunnel/cli/config.py

```diff
@@ -26,6 +33,22 @@ class Providers:
     ubuntu: providers.ubuntu.Config = field(default_factory=providers.ubuntu.Config)
     wolfi: providers.wolfi.Config = field(default_factory=providers.wolfi.Config)
 
+    # TODO: try to move this to the toplevel config and make this an init-only var
```

Comment:
I'll try to get this migrated, but if I'm out and don't get to it at least this should be moved to a sub-section here called `common` (or a better name) so future common options aren't siblings of other provider configurations.